libs/ledgerjs/packages/cryptoassets/src/currencies.ts

```diff
@@ -2567,35 +2859,6 @@ export const cryptocurrenciesById: Record<CryptoCurrencyId, CryptoCurrency> = {
       },
     ],
   },
-  juno: {
```

Comment:
This can stay IMO, a lot of currencies are defined here even if not supported in Live.